### ISSUE TITLE
Time for a new release ?

### ISSUE THREAD

**Issue by ffontaine** [steve-o/openpgm#62]
Hi,

Do you think you could tag a new release? Latest "official" release is 6 years old. I would like to use in buildroot instead of a commit tag.

Best Regards,

Fabrice

**Comment by steve-o** [comment]
So far each release has gone through a comprehensive test suite, I no
longer have that hardware hence running off commit tags.  Will test some
alternatives to push out a full release.

**Comment by acxz** [comment]
I am sure a lot of OS package maintainers would appreciate a new release as well!

**Comment by GregoryLand** [comment]
I am having a problem with ZeroMQ on Arch Linux as a result of an old issue that was fixed in this commit.  https://github.com/steve-o/openpgm/commit/ee25ff3d13f2639b4c3a42125e79f77f921c3320  I believe a new release would help get this fixed.

**Comment by acxz** [comment]
From what I understand, @GregoryLand, the Arch Linux package does not follow this repo's release cycle (although it probably would if there is a newer release), but rather it follows https://www.freshports.org/net/openpgm/ which does have the fix. However, the Arch Linux package is out of date with respect to that upstream.

In the meanwhile, I have created another package called `libpgm-git` that does follow the master branch of this git repo, but you would like an up to date Arch Linux package on openpgm.

**Comment by GregoryLand** [comment]
@acxz I ended up creating a patch from ee25ff3 to resolve the issue and submitting it.  One of the arch trusted users then took the work improved it.  It has been pushed into community for now.  Arch seems to be tracking 5-2-122 but not applying any of the patches from freshports as you have stated.  The code itself is downloading from code.google.com.  Freshports does not have an active maintainer listed for this project.  I am not the arch linux package maintainer and I am unsure why this situation is the way it is currently.  I tripped over this issue while trying to get xeus-cling to compile on Arch Linux for jupyter.  From the perspective of someone doing a drive by patching, this whole situation is confusing.  Where is the best place to get the stable version of this code?  What is considered stable?  Is it the last release on this git?  Is it the freshport patches?  Gentoo points at this repository.  I think a new release could go a long way to clearing up a lot of this confusion.
